keycloak/user-storage-provider/pom.xml

```diff
@@ -20,8 +20,8 @@
 
     <properties>
         <version.compiler.maven.plugin>3.5.1</version.compiler.maven.plugin>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
         <version.hibernate.javax.persistence>1.0.0.Final</version.hibernate.javax.persistence>
         <arquillian-graphene.version>2.5.4</arquillian-graphene.version>
         <version.jee.jaxb.api>2.3.1</version.jee.jaxb.api>
```

Comment:
Add this on super pom

Reply:
@heliocastro Do you mean all the above properties? or any specific.

Reply:
Moved it.